crates/database/src/orders.rs

```diff
@@ -54,6 +54,14 @@ pub enum BuyTokenDestination {
     Internal,
 }
 
+/// one row in the pre_interaction table
+#[derive(Clone, Debug, Default, Eq, PartialEq, sqlx::FromRow)]
+pub struct PreInteraction {
+    pub target_to: Address,
```

Comment:
uber-naming-nit: Why not just `target` to match the name of the Solidity struct? I don't feel super strong about it, so feel free to ignore.